bot/botUtils.js

```diff
@@ -33,10 +34,47 @@ exports.isAdmin = (member) => {
   return false;
 };
 
+// Checks if the member has been verified
+exports.isVerified = async (userId) => {
+  const memberData = await Member.findOne({ attributes: ['verified'], where: { discordUser: userId } });
+  if (memberData) {
+    return !!memberData.dataValues.verified;
+  }
+  return false;
+};
+
 // Update the user points in the database
 exports.getUserPointsandUpdate = async (userId, pointsToAdd) => {
-  const memberData = await Member.findAll({ attributes: ['points'], where: { discordUser: userId } });
-  await Member.update(
-    { points: memberData[0].dataValues.points + pointsToAdd },
-    { where: { discordUser: userId } });
+  const memberData = await Member.findAll({ attributes: ['points', 'verified'], where: { discordUser: userId } });
+  const { points, verified } = memberData[0].dataValues;
+  const pointsBeingAdded = parseFloat((points + pointsToAdd).toFixed(2));
+  verified ? await Member.update(
```

Comment:
I'm not sure what this line is doing.

Reply:
This line checks if they're verified in the database in order to only award points if they're verified. Do you think it needs to be changed to be more clear?

Reply:
Don't change it, but put a comment on it so that others can learn this type of conditional.